crates/wash-cli/src/app/mod.rs

```diff
@@ -125,14 +125,18 @@ pub struct PutCommand {
 
 #[derive(Args, Debug, Clone)]
 pub struct GetCommand {
-    /// The name of the application to retrieve
+    /// The name of the application to retrieve. If left empty retrieves all the applications, same as `wash app list`
```

Comment:
NIT: The first line will show up in rustdoc, so might be nice to leave a space after the first sentence here